tests/time_tests/src/timetests_helper/statistics_writer.h

```diff
@@ -46,12 +53,33 @@ class StatisticsWriter {
     }
   }
 
+  /**
+   * @brief Compute order for statistics operations.
+   */
+  void addTimer(const std::string name) {
+    tab_count++;
+    time_struct_order.push_back(name);
+  }
+
+  void deleteTimer(const std::pair<std::string, float> &record) {
+    time_structure[record.first] = std::make_pair(tab_count, record.second);
+    tab_count--;
+  }
+
   /**
    * @brief Writes provided statistics in YAML format.
    */
-  void write(const std::pair<std::string, float> &record) {
+  void write() {
     if (!statistics_file)
       throw std::runtime_error("Statistic file path isn't set");
-    statistics_file << record.first << ": " << record.second << "\n";
+    for (auto& timer: time_struct_order) {
+      std::string tabs = "";
+      for (int i = 0; i < time_structure[timer].first - 1; ++i) {
+        tabs += TAB;
```

Comment:
let's replace such `tabs` computation (`for` loop) with `std::string tabs = std::string(TAB* i, ' ')` where `TAB=4`